public/change-password.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+include "../utils/filter.php";
+include "../utils/functions.php";
+include "../utils/database.php";
+
+$message = "
+<p class=\"register-note\">
+  Note that signing up can take some time. You will get a message once password is succesfull changed
+</p>";
+
+if (isset($_POST["submit"])) { 
+  // filter all user values
+  $input["Newpassword"] = filterInputPost($_POST["Newpassword"], "Newpassword");
+  $input["Confirmpassword"] = filterInputPost($_POST["Confirmpassword"], "Confirmpassword");
+}
+  // check if any values were false, if so return an
+ //if (in_array(false, $input)) {
+ //   $message = "  <p class=\"register-error\">please double check if all fields were filled in!</p>";
+  //  if(in_array(true, $input)){
+  //      } 
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Sign up director</title>
+  <?php
+  include "../templates/head.php";
+  ?>
+  <link rel="stylesheet" href="assets/css/signup.css">
+</head>
```

Comment:
Serhan, in dit stuk tel ik meerdere malen de woorden 'register' en 'signing up'. Daarnaast ook code die uit gecomment is, waar je waarschijnlijk ook niks mee gaat doen. Het is goed dat je geen dubbel werk doet idd, maar pas het nog wel even aan naar de change-password pagina. 

Gaat voornamelijk om klasse namen, de titel van de pagina, de inhoud van de message op regel 9 etc.